README.md

```diff
@@ -1,50 +1,42 @@
-ember-template-imports
-==============================================================================
-
-This addon provides a number of different formats for using template imports
-within Ember!
+# ember-template-imports
+
+## aka First-Class Component Templates
+
+This addon began as a testing ground for various "template import" syntaxes,
+but has essentially evolved to implement the
+[First-Class Component Templates RFC](https://rfcs.emberjs.com/id/0779-first-class-component-templates/),
+which introduces `<template>` tags as a format for making component templates
+first-class participants in JavaScript and TypeScript with
+[strict mode](https://rfcs.emberjs.com/id/0496-handlebars-strict-mode/)
+template semantics. The `<template>` syntax is made available to
+JavaScript and TypeScript files with the `.gjs` and `.gts` extensions, respectively.
+
+First-class component templates address a number of pain points in today’s component
+authoring world, and provide a number of new capabilities to Ember and Glimmer users:
+
+- accessing local JavaScript values with no ceremony and no backing class, enabling much
+  easier use of existing JavaScript ecosystem tools, including especially styling
+  libraries—standard [CSS Modules](https://github.com/css-modules/css-modules)
+  will “just work,” for example
+- authoring more than one component in a single file, where colocation makes sense—and
+  thereby providing more control over a component’s public API
+- likewise authoring locally-scoped helpers, modifiers, and other JavaScript functionality
+
+Here is an example of a `.gjs` / `.gts` component file, which demonstrates how other
+components (e.g. `MyComponent`) can be imported using JS/TS `import` syntax, and then
+rendered within the embedded `<template>` tag:
 
 ```js
 import MyComponent from './my-component';
 
 <template>
+  <p>
+    Rendering MyComponent below:
+  </p>
   <MyComponent/>
 </template>
 ```
 
-Template imports are an upcoming feature in Ember. Like Glimmer components, the
```

Comment:
I'm fine with removing this particular section, but let's leave the sections *after* it.